InMobiAdapter/src/main/java/com/chartboost/mediation/inmobiadapter/InMobiAdapter.kt

```diff
@@ -384,6 +369,41 @@ class InMobiAdapter : PartnerAdapter {
         }
     }
 
+    /**
+     * Build a [JSONObject] that will be passed to the InMobi SDK for GDPR during [setUp].
+     *
+     * @param gdprConsent A Boolean indicating whether GDPR consent is granted or not.
+     *
+     * @return a [JSONObject] object as to whether GDPR  .
+     */
+    private fun buildGdprJsonObject(gdprConsent: Boolean, context: Context): JSONObject {
+        return JSONObject().apply {
+            try {
+                put(InMobiSdk.IM_GDPR_CONSENT_AVAILABLE, gdprConsent)
+                put("gdpr", if (gdprApplies == true) "1" else "0")
```

Comment:
no constant available for this?

Reply:
No constant at the moment. Here's the example from their [docs](https://support.inmobi.com/monetize/sdk-documentation/android-guidelines/overview-android-guidelines#initializing-the-sdk):

```java
    // Provide correct consent value to sdk which is obtained by User 
    consentObject.put(InMobiSdk.IM_GDPR_CONSENT_AVAILABLE, true);
    // Provide 0 if GDPR is not applicable and 1 if applicable  
    consentObject.put("gdpr", "0");
    // Provide user consent in IAB format 
    consentObject.put(InMobiSdk.IM_GDPR_CONSENT_IAB, “ << consent in IAB format >> ”);
```